Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN useradd lisk && \
     chown lisk:lisk -R /home/lisk
 USER lisk
 RUN cd /home/lisk/lisk-faucet && \
-    npm install
+    npm install --registry https://npm.lisk.io
```

Comment:
Isn't that for pre-release / devlopment only?

Reply:
yes, as i mentioned on the PR, it needs to be updated once sdk is published

Reply:
Or we could default to not using any custom registry unless `$REGISTRY_URL` is set? See https://github.com/LiskHQ/lisk-core/blob/development/docker/Dockerfile#L22

Reply:
Update not to use the registry =)